src/main/resources/assets/origins/lang/pt_br.json

```diff
@@ -158,10 +158,25 @@
   "power.origins.translucent.name": "Translucidez",
   "power.origins.translucent.description": "Sua pele é translúcida.",
 
+  "power.origins.end_spawn.name": "Nascido no Fim",
+  "power.origins.end_spawn.description": "Sua aventura começa no fim.",
+
+  "power.origins.master_of_webs.name": "Mestre das Teias",
+  "power.origins.master_of_webs.description": "Você anda perfeitamente entre as teias, e é capaz de escalar elas. Em um combate corpo a corpo contra um inimigo, ele fica preso em uma teia por um tempo. Os que não são artrópodes e ficam presos em suas teias, você é capaz de detecta-los. Você é capaz de criar uma teia de aranha a partir de uma linha.",
+
+  "power.origins.lay_eggs.name": "Ovíparo",
+  "power.origins.lay_eggs.description": "Sempre que você acorda de manhã, você botara um ovo.",
+
   "category.origins": "Origens",
-  "key.origins.primary_active": "Poder Ativo",
+  "key.origins.primary_active": "Poder Ativo(Primario)",
```

Comment:
Why is there no space between "Ativo" and "(Primario)"? Do you not have spaces in these places in your language? Same for the secondary power.

Reply:
sorry for the delay to answer you, I did not put the space because there are both types, with or without spaces, I preferred without even, and more of mine than of my language